src/gui/DatabaseWidget.h

```diff
@@ -214,7 +214,7 @@ public slots:
     void switchToDatabaseSettings();
 #ifdef WITH_XC_BROWSER_PASSKEYS
     void switchToPasskeys();
-    void switchToImportPasskey();
+    void switchToImportPasskey(bool isEntry = false);
```

Comment:
Technically we aren't switching the view for this operation (it is in a popup), so maybe rename this to just "showImportPasskey"